### DISCUSSION TITLE
Distributed Rate Limiting using SQL database (e.g. Oracle)

### DISCUSSION THREAD

**Discussion by stylianosnicoletti** [dotnet/aspnetcore#58798] [Ideas]
The Rate limiting middleware in ASP.NET Core cannot be used for distributed systems where consistency and shared rate limiting across multiple servers.

Any plan on making this feature available. 
An option to use a Distributed cache such as REDIS or SQL database.

**Comment by martincostello** [comment]
AFAIK, #41861 represents the latest state of this feature.

**Comment by stylianosnicoletti** [comment]
Thank you @martincostello! I can see this discussion is for REDIS, my suggestions is to have an SQL Database as well like Oracle

**Reply by martincostello** [reply]
Sure, but this comment is fairly generic as to just "distributed": https://github.com/dotnet/aspnetcore/issues/41861#issuecomment-2027695950
